examples/src/main/java/com/opengamma/strata/examples/finance/CalibrationCheckExample.java

```diff
@@ -198,7 +198,7 @@ public static void performance_calibration_pricing() {
         .flatMap(defn -> defn.getNodes().stream())
         // IborFixingDeposit is not a real trade, so there is no appropriate comparison
         .filter(node -> !(node instanceof IborFixingDepositCurveNode))
-        .map(node -> node.trade(VAL_DATE, MarketData.builder().addValuesById(quotes).build()))
+        .map(node -> node.trade(VAL_DATE, ImmutableMarketData.builder(VAL_DATE).addValuesById(quotes).build()))
```

Comment:
Neither `CalculationMarketData` nor `CalculationEnvironment` have factories. It also seemed to tie the interface to one specific implementation, now that there are two.